content/01.abstract.md

```diff
@@ -1,13 +1,20 @@
+## Highlights
+<!-- 3–4 bullet points of no more than 85 characters in length, including spaces,  -->
+- proportion of women honorees was similar to that of the field
+- scientists of East Asian origin have been underrepresented among honorees
+- disparities arise partly, but not exclusively, from geography
+- honorees with an affiliation in the US were overrepresented by a factor of 2.0
```

Comment:
```suggestion
- honorees with an affiliation in the United States were overrepresented by a factor of 2.0
```
Not sure if they will require this 🤷

Reply:
This is 89 characters, over the 85 limit...